lib/src/sdk/trace/exporters/collector_exporter.dart

```diff
@@ -68,7 +66,8 @@ class CollectorExporter implements sdk.SpanExporter {
       for (final attr in il.key.attributes.keys) {
         attrs.add(pb_common.KeyValue(
             key: attr,
-            value: _attributeValueToProtobuf(il.key.attributes.get(attr))));
+            value:
+                _attributeValueToProtobuf(il.key.attributes.get(attr) ?? '')));
```

Comment:
Is this a logic change? previously it's possible to get a null back from the attributes.get call .. and now you're guaranteed to get at least an empty string. I know that's the intention to make it non-nullable, but is that OK? Will it change any behavior?

Reply:
Not really, attributes is a map under the hood, so it's more to get the analyzer to shut up.  This iterates over the existing keys in the first place, so any key encountered here must exist.

Reply:
Does that mean we should use `!`?
```suggestion
                _attributeValueToProtobuf(il.key.attributes.get(attr)!)));
```